src/import-data/ImportDataDestination.ts

```diff
@@ -361,7 +359,7 @@ export const ImportDataDestination = (props: ImportDataDestinationProps): ReactN
                 renderNotice: ({ selectedBillingProject }) =>
                   h(Fragment, [
                     isProtectedData &&
-                      selectedBillingProject?.cloudPlatform === 'AZURE' &&
+                      selectedBillingProject !== undefined &&
```

Comment:
I suspect we can probably remove `selectedBillingProject` from this condition altogether.

```suggestion
```

Reply:
thought about it but decided to keep it due to the check in the tests if it was null or not. figured may be safer that way?

Reply:
I'm not sure I'd describe it as _safer_. It looks like this will only display the message once a billing project is selected. That's fine, but also not really necessary, so we could trim a tiny bit of complexity here. But either way is fine.